vagrant/global_vars.yml

```diff
@@ -15,3 +15,9 @@ install_pkgs:
 - kubeadm
 - kubelet
 - kubectl
+# Enables a custom registry
+#custom_registry: 192.168.121.1:5000
```

Comment:
If I read this right, you have to uncomment the lines without a space after the `#` sign in order to activate?
It is briefly mentioned in the readme, but
can we somehow detect/record the ip from the docker-registry-run.sh ?